src/pages/en/guides/server-side-rendering.md

```diff
@@ -132,3 +139,75 @@ export async function get({ params }) {
   });
 }
 ```
+
+In this example, a valid HTML code can be returned to render the whole page or some of its content.
+
+### Serverless Functions
```

Comment:
My only feedback really is calling this section "Serverless Functions". These are route handlers for non-GET requests, and they may or may not be deployed as serverless functions. If you build to Node.js for example, you would be using this in a non-serverless context.

Also, when deploying to a serverless environment like Vercel and Netlify, the entire app gets deployed as a single serverless function, so the individual API routes are not serverless functions themselves, as this section seems to imply.